src/dialects/mssql/query-interface.js

```diff
@@ -50,6 +50,21 @@ export class MsSqlQueryInterface extends AbstractQueryInterface {
     return this.sequelize.queryRaw(removeSql, options);
   }
 
+  /**
+    * @override
+    */
+  async bulkInsert(tableName, records, options, attributes) {
+    options = { ...options, type: QueryTypes.INSERT };
```

Comment:
you don't need to add type here, or clone the object, it's all handled by the parent method